extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -142,5 +196,56 @@ CTP payment representation:
 
 Pass the `origin key` to your front end. You might use this origin key to be able to render Component.
 
-> Note: First 2 steps are optional if origin key and payment methods has been already cached by the merchant server.
+> Note: The First 2 steps are optional if origin key and payment methods have been already cached by the merchant server.
  
+
+## Step N: Cancel or refund
+If you want to return the funds to your shopper, but are not certain whether the payment has been captured or not, use the Cancel or Refund functionality. 
```

Comment:
I don't think we need this as a Step. Maybe just another point?

Reply:
I change this with #336